polaris.shopify.com/src/components/Header/Header.module.scss

```diff
@@ -173,12 +173,12 @@
   }
 }
 
-.SideNavContainer {
+.MobileNavContainer {
   visibility: hidden;
   display: none;
-  fill: var(--text-strong);
 
   svg {
+    fill: var(--text-strong);
```

Comment:
This could probably just be `var(--text)`. Right now it's very stark compared to the text.